adapters/ipmi_generic/conf/device.properties

```diff
@@ -0,0 +1,6 @@
+model.id = 14020602
```

Comment:
maybe too late, but I think that an ID based on the current date is better (ex: 20201107)